product/models.py

```diff
@@ -0,0 +1,98 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=50)
+
+    class Meta:
+        db_table = 'categories'
+
+class SubCategory(models.Model):
+    name     = models.CharField(max_length=50)
+    category = models.ForeignKey(Category, on_delete = models.SET_NULL, null = True)
+
+    class Meta:
+        db_table = 'sub_categories'
+
+class Product(models.Model):
+    name               = models.CharField(max_length=50)
+    name_en            = models.CharField(max_length=50)
+    image_url          = models.CharField(max_length=2000)
+    description        = models.TextField(null=True)
+    default_price      = models.IntegerField(null = True)
+    toasted            = models.BooleanField(null=True)
+    customization_true = models.BooleanField(null=True)
+    nutrition          = models.OneToOneField('Nutrition', on_delete = models.SET_NULL, null=True)
+    category           = models.ForeignKey(Category, on_delete = models.SET_NULL, null = True)
+    subcategory        = models.ForeignKey(SubCategory, on_delete = models.SET_NULL, null = True)
+    product_ingredient = models.ManyToManyField('Ingredient', through='ProductIngredient')
+
+    class Meta:
+        db_table = 'products'
+
+    def __str__(self):
+        return self.name
+
+class Nutrition(models.Model):
+    size_g          = models.IntegerField(null = True)
+    calories_kcal   = models.IntegerField(null = True)
+    sugar_g         = models.IntegerField(null = True)
+    protein_g       = models.IntegerField(null = True)
+    saturated_fat_g = models.IntegerField(null = True)
+    sodium_g        = models.IntegerField(null = True)
+
+    class Meta:
+        db_table = 'nutritions'
+
+class IngredientCategory(models.Model):
+    name = models.CharField(max_length = 50)
+
+    class Meta:
+        db_table = 'ingredient_categories'
+
+class Ingredient(models.Model):
+    name                        = models.CharField(max_length=50)
+    image_url                   = models.CharField(max_length=2000)
```

Comment:
```suggestion
    image_url                   = models.CharField(max_length=2000)
```

URL의 경우, CharField 가 아닌 URLField를 추천드립니다.